### ISSUE TITLE
Instructions to set path to $HOME/.local/share/neovim instead of $HOME/.localshare/neovim

### ISSUE THREAD

**Issue by 3kyro** [MordechaiHadad/bob#20]
System: Fedora 35
When using `bob use v0.70` the output is :
````
Apr 24 22:20:15.557  INFO Please add /home/kyro/.local/share/neovim to PATH
Apr 24 22:20:15.557  INFO You can now use v0.7.0!
````
However this won't work as the nvim executable is found at `$HOME/.local/share/neovim/bin`

**Comment by MordechaiHadad** [comment]
Thanks for catching this mistake! I will update the message tomorrow.

**Comment by MordechaiHadad** [comment]
Fixed with 7defb8b8bd3bce5bb79a032944f2bd97723a7212 
Will make a new release in a few hours
